### PR TITLE
Add CODEOWNERS for cardano-node and packages therein

### PR DESCRIPTION
**Pull request by Jimbo4350**

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -77,4 +83,4 @@ _sources/flat     @input-output-hk/plutus-core
 _sources/int-cast @input-output-hk/plutus-core
 _sources/moo      @input-output-hk/cardano-ledger
 # Need a node team
-# _sources/ekg-json   
+# _sources/ekg-json
```
Comment (michaelpj) [inline CODEOWNERS:86]: here also

File: CODEOWNERS [hunk 2]
```diff
@@ -1,37 +1,43 @@
 # cardano-node
-# No team?
+_sources/cardano-api             @input-output-hk/release
+_sources/cardano-cli             @input-output-hk/release
+_sources/cardano-client-demo     @input-output-hk/release
+_sources/cardano-node            @input-output-hk/release
+_sources/cardano-submit-api      @input-output-hk/release
+_sources/cardano-testnet         @input-output-hk/release
```
Comment (michaelpj) [inline CODEOWNERS:7]: My understanding is that the release team is going to manage ledger and network also? So should we add them to those packages too?

**Review by michaelpj — APPROVED** [review]

**Comment by michaelpj** [comment]
Added the team to the repo.

File: CODEOWNERS (hunk 1 above)
Comment (Jimbo4350) [reply CODEOWNERS:86]: @kosyrevSerge this should fall under your team

File: CODEOWNERS (hunk 2 above)
Comment (Jimbo4350) [reply CODEOWNERS:7]: You'll need to ask @disassembler

File: CODEOWNERS (hunk 1 above)
Comment (deepfire) [reply CODEOWNERS:86]: @Jimbo4350, I can't figure out where the `ekg-json` is coming from.  It's not used directly by any node package.  It's also not hosted by `input-output-hk`.

From my point of view, I have no idea why we need it.

Anyone?

Comment (michaelpj) [reply CODEOWNERS:86]: We have issues for all patched packages: https://github.com/input-output-hk/cardano-haskell-packages/issues/56

You can see where it comes from here: https://github.com/input-output-hk/cardano-haskell-packages/blob/main/_sources/ekg-json/0.1.0.7.0.0.0.0.1/meta.toml#L2

We just took it from the `cardano-node` `cabal.project`, I believe it's a transitive dependency.

Comment (andreabedini) [reply CODEOWNERS:86]: ekg-json is a dependency of ekg if that's what you are asking.

**Comment by michaelpj** [comment]
Okay, we can leave the other things to resolve later.

File: CODEOWNERS (hunk 1 above)
Comment (michaelpj) [reply CODEOWNERS:89]: I think it's indirect via `cardano-tracer`?

**Pull request merged**